qdrant_client/fastembed_common.py

```
@@ -7,24 +7,268 @@
+            model_name (str): The name of the model to check.
+
+        Returns:
+            bool: True if the model is supported, False otherwise.
+        """
+        if model_name.lower() in cls._TEXT_MODELS:
+            return True
+        # update cached list in case custom models were added
+        cls._TEXT_MODELS = {model.lower() for model in cls.list_text_models()}
+        if model_name.lower() in cls._TEXT_MODELS:
+            return True
+        return False
```

Ruff (0.11.9)

Return the condition `model_name.lower() in cls._TEXT_MODELS` directly

Replace with `return model_name.lower() in cls._TEXT_MODELS`

(SIM103)